src/utils/time/index.js

```diff
@@ -20,24 +20,24 @@ export const calculateTime = (time, formatForChatTab = false) => {
   }
 
   if (minutes < 60) {
-    return minutes === 0 ? '1m ago' : `${minutes}m ago`;
+    return minutes === 0 ? '1m' : `${minutes}m`;
   }
 
   if (hours < 24) {
-    return `${hours}h ago`;
+    return `${hours}h`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the 'ago' suffix has been removed which could lead to confusion. It's recommended to keep it.
```diff
- return `${hours}h`;
+ return `${hours}h ago`;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->